src/main/proto/ga4gh/variant_service.proto

```diff
@@ -133,6 +133,46 @@ message GetVariantRequest {
   string variant_id = 1;
 }
 
+// ******************  /calls  *********************
+// This request maps to the body of `POST /calls/search` as JSON.
+message SearchCallsRequest {
```

Comment:
would be good to document which fields in search request are required vs optional.

Is there any reason to leave the array of call in a variant with this endpoint?